scripts/paramsestimator/binfhe_params_helper.py

```diff
@@ -7,11 +7,10 @@
 import sys
 import os
 import io
-from time import sleep
-#import paramstable as stdparams
+#from time import sleep
 import paramstable as stdparams
 
-syspath = "/home/sara/lattice-estimator"
+syspath = "/home/cpascoe/ops5g/lattice-estimator"
```

Comment:
let us make this more generic and add a comment explaining what this path is used for

Reply:
I suggest we remove this altogether and instruct users to add the lattice-estimator path to PYTHONPATH

Reply:
Agreed